packages/dd-trace/test/appsec/iast/analyzers/unvalidated-redirect-analyzer.spec.js

```diff
@@ -3,15 +3,23 @@
 const { expect } = require('chai')
 const proxyquire = require('proxyquire')
 const overheadController = require('../../../../src/appsec/iast/overhead-controller')
-const { HTTP_REQUEST_HEADER_VALUE, HTTP_REQUEST_PARAMETER } =
+const {
+  HTTP_REQUEST_HEADER_VALUE,
+  HTTP_REQUEST_PARAMETER,
+  HTTP_REQUEST_PATH,
+  HTTP_REQUEST_PATH_PARAM
+} =
   require('../../../../src/appsec/iast/taint-tracking/source-types')
```

Comment:
I think that we don't have any problem now with line length.
```suggestion
} = require('../../../../src/appsec/iast/taint-tracking/source-types')
```